WIP: Frontity flow component

michalczaplinski opened the pull request:
Desktop: 
<img width="870" alt="Screenshot 2020-04-12 at 21 19 16" src="https://user-images.githubusercontent.com/5417266/79086886-45f0bc00-7d03-11ea-9842-0c6483432819.png">

Mobile: 

<img width="339" alt="Screenshot 2020-04-12 at 21 19 55" src="https://user-images.githubusercontent.com/5417266/79086920-5d2fa980-7d03-11ea-8a35-52ab29e10ba7.png">

frontibotito commented:
Deploy preview for _website_ ready!

Built with commit 145e6e08c993fc6b6c3825d17d16080eb2123e51

https://frontity-org-i3wxz4nsk.now.sh

michalczaplinski commented:
@SantosGuillamot 

There are still a couple of things missing, but I'm not sure if you want to merge this first or to fix all the issues before we merge:

1. The component should have a greyish background: `rgba(193,197,222,0.2)` but I wasn't sure if this should be hardcoded or if we want to change this in WP somehow
2. I've noticed that the buttons on this design are different from the default buttons:
<img width="323" alt="Screenshot 2020-04-03 at 18 43 10" src="https://user-images.githubusercontent.com/5417266/78412953-fad7fa00-75da-11ea-83f5-5f7b368c4f2d.png">

I think I ll just have to change those buttons to regular react buttons with custom styling using `styled`

3. The padding and margins are not right on multiple components. Do we fix it now or later?

michalczaplinski commented:
@SantosGuillamot Hey, I have updated the component substantially and fixed the button styles, but it's still WIP though because I gotta fix some of the padding and add the animations.

SantosGuillamot commented:
Great, it looks really nice so far!

DAreRodz commented on packages/frontity-org-theme/src/index.ts:
```diff
@@ -45,6 +50,7 @@ const frontityOrg: FrontityOrg = {
         white: "#ffffff",
       },
       templates: ["fixed-header", "header", "footer", "newsletter"],
+      tabNumber: 1, // Used in the frontity-flow section of the homepage
```
I would use here a more descriptive name (something with `flowSection`, maybe?).

michalczaplinski replied:
yeah, I thought the same, actually 😅 I used `activeFlowSectionTab` which is a bit verbose but it's better, because the "section" alone refers to the section of the website like "frontity flow" section or "headless flow" section, etc.

DAreRodz commented on packages/frontity-org-theme/src/processors/frontity-flow-items.tsx:
```diff
@@ -0,0 +1,94 @@
+      }
+    `}
+  >
+    {Array.from({ length: 4 }, (_, i) => (
```
I didn't know this way of initializing arrays. Nice. :+1:

luisherranz commented:
Looks awesome guys 👏

luisherranz commented:
@michalczaplinski may I ask how you were able to merge a WIP pull request?
Didn't the new GitHub action prevent the merge? Do I need to fix it maybe?

michalczaplinski commented:
Oh, I had to use the "Use the administrator privilege to merge" but I assumed that it was a UI glitch because both David and Mario gave had given an approval already. I

I guess it was because the PR still had the `WIP:` in the name, right?

luisherranz commented:
Yes, that was the reason.
